src/field_theory/finite/polynomial.lean

```diff
@@ -173,7 +173,7 @@ calc vector_space.dim K (R σ K) =
     quotient.sound ⟨equiv.arrow_congr (equiv.refl σ) (equiv.fin_equiv_subtype _).symm⟩
   ... = cardinal.mk (σ → K) :
   begin
-    refine (trunc.induction_on (fintype.equiv_fin K) $ assume (e : K ≃ fin (fintype.card K)), _),
+    obtain ⟨e⟩ := fintype.trunc_equiv_fin K,
     refine quotient.sound ⟨equiv.arrow_congr (equiv.refl σ) e.symm⟩
   end
```

Comment:
you can replace the whole `begin ... end` block with
```
    quotient.sound ⟨equiv.arrow_congr (equiv.refl σ) (fintype.equiv_fin K).symm⟩
```
Here and in most places where there was an `obtain`, you can get rid of it and use directly `fintype.equiv_fin`.